src/utils.h

```diff
@@ -70,6 +71,7 @@ struct IKParams
     std::string solver_class_name;
     bool enable_counter;
     int thread_count;
+    std::minstd_rand::result_type random_seed = std::random_device()();
```

Comment:
no need for initialization here, right? The struct should not be used standalone.

Reply:
correct but `Random` is a public API that's why I added it. Anyway, will remove.

Reply:
Well the current `HEAD` seeds `Random` by default so here are probably code parts where this is needed. Also see my proposes for `lookupParam` below.